frontend/src/components/SignUp/signup_error.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './signup.css';
+import gmail_icon from './gmail_icon.png';
+import fb_icon from './fb_icon.png';
+import apple_icon from './apple_icon.png';
+import twtx_icon from './twtx_icon.webp';
+
+export function SignUpError() {
+    return (
+        <div className="signup-group">
+            <h1>SIGN UP</h1>
+            <p className='createAccount-subtext'>create account</p>
+            <form>
+                <label>Email</label>
+                    <input className='inputBox' type="text" />
+                    <p className="errorMessage"> Please enter a valid email. </p>
```

Comment:
Logic wise, how would you determine if the email is valid or not? How should we check?